### PR TITLE
Add Chinese translation

### PR DESCRIPTION
**Pull request by pedoc**
# Reference issue

- #258

## Purpose

-   Added Chinese translation

## TODO

-   [x] Read [Contributing](https://github.com/Levminer/authme/blob/main/.github/CONTRIBUTING.md)
-   [x] Read [Code of conduct](https://github.com/Levminer/authme/blob/main/.github/CODE_OF_CONDUCT.md)
-   [x] Run `npm run lint`

### CODE REVIEW

File: interface/utils/language/zh.ts [hunk 1]
```diff
@@ -0,0 +1,181 @@
+export const localeEN = {
```
Comment (Levminer) [inline interface/utils/language/zh.ts]: This should be localeZH

File: interface/utils/language/zh.ts [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+		// html dialogs
+		captureScreenTitle: "从屏幕导入",
+		captureScreenWaiting: "等待二维码...",
+		manualEntry: "配置密钥",
+		manualEntryText: "请输入名称和 2FA 密钥.",
+		manualEntryName: "名称 (必填)",
+		manualEntrySecret: "秘钥 (必填)",
+		manualEntryDescription: "说明",
+		// tutorial
+		googleAuthTutorial: ["从 Google Authenticato 导出2FA 代码: 点击屏幕左上角的菜单:转移账户->导出账户", "从屏幕截图并保存二维码,或者如果你使用的是 Android 手机,可以用另一部手机拍照.将这些照片传输到你的电脑上", "进入 Authme 的 导入 页面: 侧边栏 -> 导入", "单击“选择图片”按钮，然后选择从手机传输的图片"],
```
Comment (Levminer) [inline interface/utils/language/zh.ts]: Missed an "a" Google Authentico

**Review by Levminer — CHANGES_REQUESTED** [review]
Thanks for your work! Please fix the 2 issues I commented on.

**Comment by pedoc** [comment]
The problem has been solved, please check

**Review by Levminer — APPROVED** [review]
Thanks!

**Pull request merged**